.eslintrc.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   plugins: ['react', 'react-native', 'import', '@typescript-eslint'],
   rules: {
     'no-console': 'off',
+    'no-unused-vars': 'off',
```

Comment:
Why this change?

Reply:
I figured we could turn this off because `typescript-eslint/no-unused-vars` already checks for this.